packages/plugin-ext/src/plugin/preference-registry.ts

```diff
@@ -149,19 +157,26 @@ export class PreferenceRegistryExtImpl implements PreferenceRegistryExt {
         return configuration;
     }
 
+    // tslint:disable-next-line:no-any
```

Comment:
it makes the code barely readable to have this every couple of lines

Reply:
What I can do is disable the `no-any` flag for the entire file by placing it at the beginning of the file. I understand that it is an api file and thus we want to keep the api as generic as possible but if ever an instance does not make sense to use any, it would be the developer's responsibility to not abuse the `no-any` rule at the beginning and use the correct type.

Reply:
I'll make the changes for you @benoitf